pages/resultset.py

```diff
@@ -17,21 +17,27 @@ class ResultsetPage(Base):
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
     _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
     _result_status_locator = (By.ID, 'job-details-panel')
+    _unclassified_failure_count_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     @property
     def job_result_status(self):
         return self.selenium.find_element(*self._job_result_status_locator).text
 
+    @property
+    def verify_unclassified_failure_exists(self):
+        return self.selenium.find_element(*self.unclassified_failure_count_locator).text
```

Comment:
Rather than send back a string that needs to be compared in the test, I would suggest a property that returns a count of the unclassified results displayed on the page. Having only a brief look, could we use the `btn-orange` or `btn-red` classes to identify unclassified results? If so, we could just return the length of `find_elements` and loop clicking the button until the return value is > 0. I would also be fine with having a `has_classified_jobs` in the page object that wraps this logic up.